src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -48,6 +48,19 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         Public MustOverride Overloads Sub Accept(visitor As OperationVisitor) Implements IOperation.Accept
 
         Public MustOverride Overloads Function Accept(Of TArgument, TResult)(visitor As OperationVisitor(Of TArgument, TResult), argument As TArgument) As TResult Implements IOperation.Accept
+
+        Protected Shared Function GetBadChild(parent As BoundNode, index As Integer) As IOperation
```

Comment:
I think the name is misleading, the child isn't necessary bad, its parent is bad though. I'd use name "GetChildOfBadExpression" or something similar.